cmd/clusterctl/client/init.go

```diff
@@ -58,6 +59,12 @@ type InitOptions struct {
 	// LogUsageInstructions instructs the init command to print the usage instructions in case of first run.
 	LogUsageInstructions bool
 
+	// WaitProviders instructs the init command to wait till the providers are installed.
+	WaitProviders bool
+
+	// WaitProviderTimeout sets the timeout per provider wait installation
```

Comment:
```suggestion
	// WaitProvidersTimeout sets the timeout per provider wait installation.
```

Reply:
I will add the period at the end but I think the variable should be `WaitProviderTimeout` (singular) since the timeout defined here per provider. I will refactor the variable to match the comment (make it singular).